package.json

```diff
@@ -115,14 +125,19 @@
   },
   "peerDependencies": {
     "dotenv": ">=15",
+    "yaml": "^2.x",
     "zod": "^3.x"
   },
   "peerDependenciesMeta": {
     "dotenv": {
       "optional": true
+    },
+    "yaml": {
+      "optional": true
     }
   },
   "engines": {
     "node": ">=14.0.0"
-  }
+  },
+  "packageManager": "pnpm@8.15.9"
```

Comment:
note: an older version of pnpm is required so that the version of `pnpm-lock.yaml` won't be automatically updated.